src/Applications/RTFRevit/RTFRevit.csproj

```diff
@@ -35,19 +36,35 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="AdWindows, Version=2015.11.1.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Revit-2017.1.1-x64.Base.2.0.0\lib\net452\AdWindows.dll</HintPath>
+      <Private>False</Private>
+    </Reference>
     <Reference Include="nunit.core">
       <HintPath>..\..\..\lib\NUnit\nunit.core.dll</HintPath>
     </Reference>
     <Reference Include="nunit.core.interfaces, Version=2.6.2.12296, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\lib\NUnit\nunit.core.interfaces.dll</HintPath>
     </Reference>
-    <Reference Include="RevitAPI">
-      <HintPath>$(REVITAPI)\RevitAPI.dll</HintPath>
+    <Reference Include="RevitAPI, Version=17.0.0.0, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\..\packages\Revit-2017.1.1-x64.Base.2.0.0\lib\net452\RevitAPI.dll</HintPath>
```

Comment:
where is this nuget package from? Is this an official Autodesk distribution? Kind of odd that this is a 2019 branch of RTF but this is using a 2017 package?

Reply:
Hi @marchello2000 I'm really sorry for no reply after such a long time. But I think it's better to use $(REVITAPI) path here, the package you use from nuget website may be a little risky. If you don't want to use the installed Revit, you can add a extern folder to $(REVITAPI) like 
<REVITAPI Condition=" !Exists('$(REVITAPI)') ">$(SolutionDir)..\lib\Revit $(RevitVersionNumber)\net47</REVITAPI>. 
When CI build, you can download the RevitAPI you want to use and place them in this folder.
If you have no time for this, I would help you merge this PR and fix this problem later.

Reply:
@ZiyunShang Thanks for merging, the PR. I see you already merged the PR for `$(REVITAPI)` change, let me comment on there as I think it's more relevant